assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -86,6 +91,38 @@ function MyStories() {
     });
   }, [stories, typeaheadValue]);
 
+  const setPageSize = useCallback(() => {
+    const { innerWidth } = window;
```

Comment:
It's probably better to use here the grid container width?

Reply:
I thought that at first but the grid uses CSS media queries, so to keep them unified, we should use the window width.